### PR TITLE
done

### PR DESCRIPTION
**Pull request by Richaa2**

### CODE REVIEW

File: src/main/java/mate/jdbc/model/Manufacturer.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private String country;
+
+    public Manufacturer(Long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
+    }
+
+    @Override
+    public String toString() {
```
Comment (nacenik) [inline src/main/java/mate/jdbc/model/Manufacturer.java:15]: toString should be after other public methods

File: src/main/resources/init_db.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE `manufacturer_db` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+CREATE TABLE `manufacturers` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
+  `country` varchar(45) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
```
Comment (nacenik) [inline src/main/resources/init_db.sql:6]: you have DEFAULT CHARSET already for whole table, why do you set it up one more type?  
```suggestion
  `name` varchar(45) DEFAULT NULL,
  `country` varchar(45) DEFAULT NULL,
```

**Review by nacenik — CHANGES_REQUESTED** [review]

File: src/main/java/mate/jdbc/dao/ManufacturerDao.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    Manufacturer create(Manufacturer manufacturer);
+
+    Optional<Manufacturer> get(Long id);
+
+    List<Manufacturer> getAll();
+
+    Manufacturer update(Manufacturer manufacturer);
+
+    boolean delete(Long id);
+
```
Comment (kozhukhovsky) [inline src/main/java/mate/jdbc/dao/ManufacturerDao.java:19]: remove empty line

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            Manufacturer manufacturer = null;
+            if (resultSet.next()) {
+                manufacturer = getManufacturer(resultSet);
+            }
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't find manufacturer from DB with this id: "
+                    + id + " ", e);
+        }
+
```
Comment (kozhukhovsky) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:55]: remove empty line

File: src/main/resources/init_db.sql [hunk 5]
```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE `manufacturer_db` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+CREATE TABLE `manufacturers` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) DEFAULT NULL,
+  `country` varchar(45) DEFAULT NULL,
+  `is_deleted` tinyint NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=9 DEFAULT CHARSET=utf8mb3;
+
```
Comment (kozhukhovsky) [inline src/main/resources/init_db.sql:10]: remove empty line

**Review by kozhukhovsky — APPROVED** [review]

**Pull request closed without merging**